src/azure-cli/azure/cli/command_modules/batch/tests/latest/test_batch_mgmt_commands.py

```diff
@@ -343,6 +343,10 @@ def test_batch_application_cmd(self, resource_group, storage_account):
 
 
 class BatchMgmtByosScenarioTests(ScenarioTest):
+
+    def __init__(self, method_name, *arg, **kwargs):
+        super().__init__(method_name, *arg, random_config_dir=True, **kwargs)
```

Comment:
No need to explicitly specify `method_name`. It is part of `*arg`.

```suggestion
    def __init__(self, *arg, **kwargs):
        super().__init__(*arg, random_config_dir=True, **kwargs)
```

Reply:
Well, as this pattern is frequently used elsewhere:

https://github.com/Azure/azure-cli/blob/21b54b5a7a41865b3f8c65640186d40479461f66/src/azure-cli/azure/cli/command_modules/serviceconnector/tests/latest/test_webpp_connection_scenario.py#L23-L27

You can keep it for now.